src/components/SectionAbout/sectionAboutUs.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/index.scss';
+
+.sectionAboutUs {
+  &-title {
```

Comment:
modyfikator powinien mieć  podwójną kreseczkę, czyli &--title, to samo w pliky tsx.